.github/workflows/unit_test.yaml

```diff
@@ -1 +1,30 @@
 name: unit test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+
+jobs:
+  unit_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repo
+        uses: actions/checkout@v2
+      - name: Setup conda env
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniconda-version: "latest"
+          activate-environment: test
+          python-version: "3.8"
+      - name: Install dependencies
+        shell: bash -l {0}
+        run: |
+          conda activate test
+          conda install pytorch torchvision torchtext cpuonly -c pytorch-nightly
+          pip install -r requirements.txt
```

Comment:
Given the changes in #26, should we also add `pip install -r dev-requirements.txt` for pytest installation?

Reply:
nit: it's more explicit to use `pip3` than `pip` even though there's no ambiguity here given the conda env is created for python 3.8.

Reply:
So the tests are running without `python setup.py install` our own library?

Reply:
> Given the changes in #26, should we also add `pip install -r dev-requirements.txt` for pytest installation?

I think it should be `pip install -e .[dev]` which installs our own library as well as the dev requirements (thus replacing `python setup.py develop`)